bclientapi/bendoapi.go

```diff
@@ -23,7 +23,11 @@ var (
 func (ia *itemAttributes) GetItemInfo() (*jason.Object, error) {
 	var path = "http://" + ia.bendoServer + "/item/" + ia.item
 
-	r, err := http.Get(path)
+	req, _ := http.NewRequest("GET", path, nil)
+        if ia.token != "" { req.Header.Add("X-Api-Key", ia.token) }
```

Comment:
There is a lot of commonality making requests to the bendo server between `GetItemInfo()`, `download()`, `PostUpload()`, `createFileTransAction()`, and `getTransactionStatus()`. Any more and it will be time to make a utility function (cf. https://github.com/ndlib/bendo/blob/master/store/cow.go#L215-L224).